lib/telephony/alert_sender.rb

```diff
@@ -71,18 +71,18 @@ def log_response(response, context:)
         channel: :sms,
         context: context,
       }
-      output = response.to_h.merge(extra).to_json
-      Telephony.config.logger.info(output)
+      output = response.to_h.merge(extra)
+      Telephony.config.log_info(event: output)
```

Comment:
I almost feel like this belongs as `Telephony.log` instead of `Telephony.config.log`? but otherwise same same

Reply:
Ah yeah, good point, moved in a484e60